regen/embed.pl

```diff
@@ -132,14 +140,14 @@ sub open_print_header {
 
 	die_at_end "For '$plain_func', M flag requires p flag"
 					    if $flags =~ /M/ && $flags !~ /p/;
-	die_at_end "For '$plain_func', C flag requires one of [pim] flag"
-					    if $flags =~ /C/ && $flags !~ /[ibmp]/;
-	die_at_end "For '$plain_func', X flag requires p or i flag"
-					    if $flags =~ /X/ && $flags !~ /[ip]/;
+	die_at_end "For '$plain_func', C flag requires one of [bmIip] flags"
```

Comment:
This is an unrelated bug fix, that the diagnostic didn't match the check.  I don't know that it matters here, but normally this change should go in a separate commit, so that if this one, for whatever reason, gets reverted, the unrelated bug fix stays.

Reply:
Also, should there be a check that i and I aren't both specified?  Maybe other illegal combos?

Reply:
I think I agree, I'll move it to a separate commit. And yes, adding that check makes sense too.